packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -260,6 +286,29 @@ const ActivityDetailsSidebar = (props: Props) => {
                 value={selectedUsers}
                 multiple={false}
               />
+
+              {showOrgPicker && (
+                <>
+                  <div className='text-gray-700 my-4 text-sm font-semibold'>Organization</div>
+                  <Select
+                    onValueChange={(orgId) => setSelectedOrgId(orgId)}
+                    value={newOneOnOneTeamOrgId}
+                  >
+                    <SelectTrigger className='bg-white'>
+                      <SelectValue />
+                    </SelectTrigger>
+                    <SelectContent>
+                      <SelectGroup>
+                        {viewerOrganizations.map((org) => (
```

Comment:
-1 This should be `mutualOrgs`, no? Or if we want to allow users to create a team on an org the other user isn't a part of, we should _always_ show this org picker